### ISSUE TITLE
PowerPC assembler is broken

### ISSUE THREAD

**Issue by zachriggle** [Gallopsled/pwntools#521]
```
>>> context.arch='powerpc'
>>> asm('li r3,0')
<errors>
>>> asm('li 3,0')
'8`\x00\x00'
>>> disasm(asm('li 3,0'))
'   0:   38 60 00 00     li      r3,0'
```

This is because the GNU gas assembler for PowerPC doesn't expressly recognize r# as a register.  General use of PowerPC includes the contents of `ppc_asm.tmpl` which is basically a bunch of `#define r3 3`.

We should add the contents of this file to our preprocessor includes for PowerPC.

**Comment by zachriggle** [comment]
This was fixed in Binjitsu with 1ac87052c2338ec5e72524679ebb8c141ed0a086

**Comment by Idolf** [comment]
Fixed by #636.
